### DISCUSSION TITLE
dont move scroll to specific anchor in Persian language

### DISCUSSION THREAD

**Discussion by mohammad2223** [requarks/wiki#3977] [Error / Bug Report]
Hi
in Persian language link when I use chrome browser, the page don't move to anchor location when loaded
 check blow link :
http://wiki.chapyar.com/fa/شروع_به_کار/نصب#فعال-سازی-نرم-افزار

**Comment by mostafa8026** [comment]
fix proposed:
https://github.com/Requarks/wiki/pull/4244

**Reply by mohammad2223** [reply]
Thanks buddy
